engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/views/product-repositories.html

```diff
@@ -23,9 +23,19 @@
     </div>
 
     <div data-block="actions">
-      <div bst-modal="removeSelectedRepositories()">
-        <div data-block="modal-header" translate>Remove {{ detailsTable.numSelected  }} Repositories?</div>
-        <div data-block="modal-body" translate>Are you sure you want to remove the {{ detailsTable.numSelected }} repositories selected?</div>
+      <div bst-modal="removeSelectedRepositories()" model="detailsTable">
+        <div data-block="modal-header" 
+             translate 
+             translate-n="detailsTable.numSelected" 
+             translate-plural="Remove {{ detailsTable.numSelected }} repostories?">
+          Remove {{ detailsTable.numSelected  }} repository?
```

Comment:
That's fair, I'll change all of these to not show the total when there is only one.